thalassa/normalization.py

```diff
@@ -19,19 +19,29 @@ class THALASSA_FORMATS(enum.Enum):
     UNKNOWN = "UNKNOWN"
     ADCIRC = "ADCIRC"
     SCHISM = "SCHISM"
+    TELEMAC = "TELEMAC"
```

Comment:
will thalassa be able to read  "vanilla" Telemac output? Or will the output need to be post-processed by PyPoseidon? 

If it is the latter, then we should perhaps name the format as `PYPOSEIDON-TELEMAC` or something.

Reply:
This PR is only for the "vanilla" Telemac selafin (It can be also other extension but with the same format as the `xarray-selafin` output). 

This [commit](https://github.com/ec-jrc/pyPoseidon/commit/8796f49df2352ab7762610044a0f64c5a97aec58) in my fork of pyPoseidon should already take care about Thalassa conventions. I will also add it to the `telemac` branch on the official repo.